### PR TITLE
[FIXES-#1997] Reverts documentation about including the FindSeqAn.cmake.

### LINKED ISSUE #1997: Wrong example in doc for cmake projects in manual
In the [example](http://seqan.readthedocs.io/en/develop/Infrastructure/Use/FindSeqAnCMake.html#building-the-project) for using the sedan-config.cmake module, we have an outdated example using the old -DCMAKE_MODULE_PATH=~/devel/seqan/util/cmake

### PR DESCRIPTION
**Pull request by rrahn**
fixes #1997 

Reverts 44159bef2c793dbf75b65a8ff8a8ed3e3abc7713

### CODE REVIEW

**Comment by h-2** [comment]
Well the green stuff is not really correct now either :expressionless: 
The thing is that the module is never in a system-wide location. The config might be! So for system-wide detection the config is used, and for local detection "probably the module" and "possibly the config".
It's a :boom:'up thing.

**Comment by rrahn** [comment]
This is all so 😞 ing

**Comment by rrahn** [comment]
rebuild

**Comment by rrahn** [comment]
@h-2 updated the docs

File: manual/source/Infrastructure/Use/FindSeqAnCMake.rst [hunk 1]
```diff
@@ -77,26 +77,54 @@ First you should create a build directory, i.e. for cmake-builds everything happ
...
+      -DCMAKE_PREFIX_PATH="$HOME/local/seqan"
+
 
-If you instead did a full git checkout to your home-directory in the previous step, it might look like this:
+SeqAn from GitHub clone
+~~~~~~~~~~~~~~~~~~~~~~~
+
+In case you obtained SeqAn from a git clone, we still recommend to use the method described above and set the ``CMAKE_PREFIX_PATH`` to your SeqAn clone.
+But in addition you also have to specify the ``SEQAN_INCLUDE_PATH`` variable to find the SeqAn headers. 
+Assume you have cloned SeqAn into ``~/devel/seqan``, then your setup could look as the following:
```
Comment (h-2) [inline manual/source/Infrastructure/Use/FindSeqAnCMake.rst]: I don't understand why this is different from the paragraph above. Why do they not need to set the INCLUDE_PATH?

File: manual/source/Infrastructure/Use/FindSeqAnCMake.rst [hunk 2]
```diff
@@ -77,26 +77,54 @@ First you should create a build directory, i.e. for cmake-builds everything happ
...
+   
+   # cmake ../../my_project \
+      -DCMAKE_PREFIX_PATH="$HOME/devel/seqan/util/cmake" \
+      -DSEQAN_INCLUDE_PATH="$HOME/devel/seqan/include"
+
+Backwards compatibility
+~~~~~~~~~~~~~~~~~~~~~~~
 
+Before SeqAn 2.3 we used the module mode to setup SeqAn as an external project.
+To allow backwards compatibility we added a symlink from the ``FindSeqAn.cmake`` to ``seqan-config.cmake``.
```
Comment (h-2) [inline manual/source/Infrastructure/Use/FindSeqAnCMake.rst]: It's not a symlink anymore, but just includes the other cmake file.

File: manual/source/Infrastructure/Use/FindSeqAnCMake.rst (hunk 1 above)
Comment (rrahn) [inline manual/source/Infrastructure/Use/FindSeqAnCMake.rst]: They have to set the include path. Read sentence at line 107. But in addition ...
Simply stating that the approach with selecting the PrefixPath is preferred over the module_path option for future compatibility.
There is also a code example.

File: manual/source/Infrastructure/Use/FindSeqAnCMake.rst (hunk 2 above)
Comment (rrahn) [inline manual/source/Infrastructure/Use/FindSeqAnCMake.rst]: Ok, but does it make any difference to the user? I will write call it redirect.

**Pull request merged**